### PR TITLE
Afrotech global summit

### PR DESCRIPTION
**Pull request by misheck12**
Hello Code Reviewer, I have completed the Milestone by creating a clone of the conference website. The website includes key features such as media queries, dynamic pages, page links, and interactions. Please find my presentation video here: https://www.loom.com/share/c7fdbb1a86134bd499e3a6c7affdc5d6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,36 @@
...
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
```
Comment (CollinsTatang) [inline README.md:18]: - [x] Kindly add a live demo link to your project README file.
- [x] Please use GitHub Pages to deploy the web pages.

File: README.md [hunk 2]
```diff
@@ -64,10 +44,6 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
```
Comment (CollinsTatang) [inline README.md:47]: - [x] Please do your best to add Acknowlegementts section  given appropriate credit to [Cindy Shin](https://www.behance.net/adagio07)

File: about.html [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    </main>
+    <footer>
+      <div class="logo">
+        <span id="logo-span">
+         <a href="index.html"><i class="fa-brands fa-artstation fa-2x "></i></a>
+        </span>
+      </div>
+      <p id="footer-p" class="foot">
+        Afrotech's 2022 global Summit. All &copy; Rights Reserved</p>
+    </footer>
```
Comment (CollinsTatang) [inline about.html:134]: - [x] Please both `Home` and `About` page should have the same footers.

File: speakers.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
+const speakersSection = document.querySelector('#speakers');
+
+const speakersObject = [
+  {
+    name: 'H.E Hakainde Hichilema',
+    about: 'Current president of Zambia',
+    history:
+      'He has been in opposition for 20 years failed 4 times. he owns shares in big global companies.',
```
Comment (CollinsTatang) [inline speakers.js:9]: - [x] Please your object should contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker. 
````
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
````

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @misheck12, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
### Highlight
- No linter errors 
- Good commit message 
- Descriptive PR message

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (misheck12) [reply README.md:18]: Hey @CollinsTatang , thank you for bringing attention to the need for a live demo link in the project README file. I have now added the instructions and deployed the web pages using GitHub Pages. Please let me know if there's anything else I can do to improve the project.

File: README.md (hunk 2 above)
Comment (misheck12) [reply README.md:47]: Thank you for highlighting the need for an Acknowledgements section in the project. I have now added it and given appropriate credit to Cindy Shin. I'm very sorry for any inconvenience caused by the oversight. I will make sure to double-check for such in the future.

File: about.html (hunk 3 above)
Comment (misheck12) [reply about.html:134]: Thank you for highlighting the footer issue on the Home and About pages. I have now made sure that both pages have the same footer. I'm very sorry for any confusion caused by the inconsistency. I'll make sure to double-check for that in the future.

File: speakers.js (hunk 4 above)
Comment (misheck12) [reply speakers.js:9]: Thank you for bringing attention to the missing information in my object for the speaker. I have now added the name, role, description, and image url as specified. I apologize for any confusion caused by the oversight. I understand the importance of providing all the necessary information to generate the dynamic element. And in regards to the photo, I was trying to keep it simple for styling purposes, but I see now that it's important to have it in the object.

File: README.md (hunk 2 above)
Comment (Whoistolu) [reply README.md:47]: All good now 👍🏽

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md:18]: All good now 👍🏽

File: about.html (hunk 3 above)
Comment (Whoistolu) [reply about.html:134]: All good now 👍🏽

File: speakers.js (hunk 4 above)
Comment (Whoistolu) [reply speakers.js:9]: - [ ] [REQUIRED] As recommended by the previous reviewer, kindly ensure that the object contains all the information needed to generate the dynamic element.  Please remove the images and populate them using Javascript.

```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```

File: index.html [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+          <i class="fa-brands fa-artstation fa-2x"></i>
+        </span>
+        <div id="logo-name">
+          <span>Afrotech</span>
+        </div>
+      </div>
+      <p id="footer-p">
+       Afrotech Summit 2021. All &copy; Rights Reserved
+      </p>
+    </footer>
```
Comment (Whoistolu) [inline index.html:142]: - [ ] [OPTIONAL] Kindly ensure to reduce the space in between the logo and the text on the footer of the mobile for both the `Homepage` and the `About page`.

Screenshot from 2022-03-11 21-07-02

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @misheck12 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by misheck12** [comment]
Hey  @CollinsTatang , @Whoistolu Thank you for bringing attention to the spacing issue on the footer of the mobile version of the Homepage and About page. I have now reduced the space between the logo and the text to ensure proper alignment and user experience. I apologize for any inconvenience caused by the oversight. I will make sure to double-check for such issues in the future.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+        </ul>
+        <div id="campaign">
+          <span>Campaign</span>
+        </div>
+      </div>
+    </nav>
+    <header class="menu">
+      <span>
+        <i class="fas fa-bars fa-2x" id="menu"></i>
+      </span>
```
Comment (elmar8287) [inline about.html:53]: - [x] [OPTIONAL] Here on the `about.html` page, you can see your mobile menu has issue with the `x` (close) icon. It is disappeared when the small screens size changes. I am sure you can easily fix it 👍 
Screen Shot 2022-03-12 at 13 59 19

> The animation includes, also:

**Review by elmar8287 — APPROVED** [review]
Hi, 

You covered all suggestions from the previous reviewers 👍 I have some OPTIONAL suggestions also, but you should know, that your project is...

# APPROVED  👍 🥇 

It's time to merge it :shipit: after fixing OPTIONAL suggestion.

- [ ] Please, consider fixing the MIT.md link - it is actually broken.
- [ ] Please, consider my OPTIONAL comment under the review 👍 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 6 above)
Comment (misheck12) [reply about.html:53]: Thanks working on this now

**Pull request merged**